rules/mastg-android-autoverify-missing.yml

```
@@ -0,0 +1,24 @@
+rules:
+- id: android-autoverify-missing
+  languages:
+    - xml
+  severity: WARNING
```

The severity should come above language attribute

```suggestion
  severity: WARNING
  languages:
    - xml
```